src/Console/AddCommand.php

```diff
@@ -138,24 +138,33 @@ private function getProperty(
                 new Question("$desc: ", $default)
             );
         }
-        //$value = $value ?: "";
         return $value;
     }
 
     private function setPayerNumber(
         string $value,
         DonorBuilder $donorBuilder
     ) {
-        $donorBuilder->setPayerNumber($value);
+        if (is_numeric($value) && strlen($value) <= 16) {
+            $donorBuilder->setPayerNumber($value);
+        } else {
+            throw new \Exception('Payer number must be numerical, and max 16 digits');
+        }
     }
 
     private function setAccount(
         $value,
         DonorBuilder $donorBuilder,
         AccountFactory $accountFactory
     ) {
-        $newAccount = $accountFactory->createAccount($value);
-        $donorBuilder->setAccount($newAccount);
+        //TODO: the accountFactory seems quite liberal in what values it
```

Comment:
I test-ran the application, adding account numbers like 'as235234dv'. An account was added as '235 23-4'.